client/state/stats/plan-usage/selectors.ts

```diff
@@ -0,0 +1,15 @@
+import { get } from 'lodash';
+import { PlanUsage } from 'calypso/my-sites/stats/hooks/use-plan-usage-query';
+
+import 'calypso/state/stats/init';
+
+export function getPlanUsageBillableMonthlyViews( state: object, siteId: number ) {
```

Comment:
We've already got this calculated from the backend if I'm not mistaken?

https://github.com/Automattic/wp-calypso/pull/92137

Reply:
ah sorry just saw your comment on L8 😄

Reply:
Yes, but I want to confirm the calculation since the same views count array would yield different results. For example, with `[3, 0, 3]`, the `billable_monthly_views` would be `0`, and the calculation here would be `3`. Which one was the original thinking? 🤔

Reply:
Because extreme cases occur rarely, the calculated number other than that would not have a significant impact. For consistency, let's use `billable_monthly_views` here. Does it make sense? @kangzj